postprocessing/resstockpostproc/resources/publication/sdr_column_definitions.csv

```diff
@@ -1,759 +1,801 @@
+Input,build_existing_model.clothes_washer_usage_level,,in.clothes_washer_usage_level,in.clothes_washer_usage_level,,,,,,,,,:ref:`clothes_washer_usage_level`
+Input,build_existing_model.cooking_range,,in.cooking_range,in.cooking_range,,,,,,,,,:ref:`cooking_range`
+Input,build_existing_model.cooking_range_usage_level,,in.cooking_range_usage_level,in.cooking_range_usage_level,,,,,,,,,:ref:`cooking_range_usage_level`
+Input,build_existing_model.cooling_unavailable_days,,in.cooling_unavailable_days,in.cooling_unavailable_days,,,,,,,,,:ref:`cooling_unavailable_days`
+Input,build_existing_model.cooling_setpoint,,in.cooling_setpoint,in.cooling_setpoint,,,,,,,,,:ref:`cooling_setpoint`
+Input,build_existing_model.cooling_setpoint_has_offset,,in.cooling_setpoint_has_offset,in.cooling_setpoint_has_offset,,,,,,,,,:ref:`cooling_setpoint_has_offset`
+Input,build_existing_model.cooling_setpoint_offset_magnitude,,in.cooling_setpoint_offset_magnitude,in.cooling_setpoint_offset_magnitude,,,,,,,,,:ref:`cooling_setpoint_offset_magnitude`
+Input,build_existing_model.cooling_setpoint_offset_period,,in.cooling_setpoint_offset_period,in.cooling_setpoint_offset_period,,,,,,,,,:ref:`cooling_setpoint_offset_period`
+Input,build_existing_model.corridor,,in.corridor,in.corridor,,,,,,,,,:ref:`corridor`
+Input,build_existing_model.county,,in.county,in.county,,,,,,,,,:ref:`county`
+Input,build_existing_model.county_and_puma,,in.county_and_puma,in.county_and_puma,,,,,,,,,:ref:`county_and_puma`
+Input,build_existing_model.county_metro_status,,in.county_metro_status,in.county_metro_status,,,,,,,,,:ref:`county_metro_status`
+Input,build_existing_model.custom_state,,in.custom_state,in.custom_state,,,,,,,,,:ref:`custom_state`
+Input,build_existing_model.dehumidifier,,in.dehumidifier,in.dehumidifier,,,,,,,,,:ref:`dehumidifier`
+Input,build_existing_model.dishwasher,,in.dishwasher,in.dishwasher,,,,,,,,,:ref:`dishwasher`
+Input,build_existing_model.dishwasher_usage_level,,in.dishwasher_usage_level,in.dishwasher_usage_level,,,,,,,,,:ref:`dishwasher_usage_level`
+Input,build_existing_model.door_area,,in.door_area,in.door_area,,,,,,,,,:ref:`door_area`
+Input,build_existing_model.doors,,in.doors,in.doors,,,,,,,,,:ref:`doors`
+Input,build_existing_model.duct_leakage_and_insulation,,in.duct_leakage_and_insulation,in.duct_leakage_and_insulation,,,,,,,,,:ref:`duct_leakage_and_insulation`
+Input,build_existing_model.duct_location,,in.duct_location,in.duct_location,,,,,,,,,:ref:`duct_location`
+Input,build_existing_model.eaves,,in.eaves,in.eaves,,,,,,,,,:ref:`eaves`
+Input,build_existing_model.electric_panel_service_rating,amp,in.electric_panel_service_rating,in.electric_panel_service_rating,,,,,,,,,:ref:`electric_panel_service_rating`
+Input,build_existing_model.electric_panel_service_rating_bin,amp,in.electric_panel_service_rating_bin,in.electric_panel_service_rating_bin,,,,,,,,,:ref:`electric_panel_service_rating_bin`

```

Comment:
<img width="1725" height="74" alt="image" src="https://github.com/user-attachments/assets/f35597cf-4fa9-4b68-a5aa-276e5cd54a16" />
Two in. columns missing Published Annual Unit and conversion factor.

Reply:
Thanks, the bin version has no conversion because it's str, but I'll add it to the numeric version.